tests/backend/integration/ProcedureMapSettingApiTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace backend\integration;
+
+use demosplan\DemosPlanCoreBundle\DataFixtures\ORM\TestData\LoadUserData;
+use demosplan\DemosPlanCoreBundle\DataGenerator\Factory\Procedure\ProcedureFactory;
+use demosplan\DemosPlanCoreBundle\DataGenerator\Factory\Procedure\ProcedureSettingsFactory;
+use demosplan\DemosPlanCoreBundle\Entity\Procedure\Procedure;
+use demosplan\DemosPlanCoreBundle\Entity\Procedure\ProcedureSettings;
+use demosplan\DemosPlanCoreBundle\Entity\User\User;
+use Lexik\Bundle\JWTAuthenticationBundle\Services\JWTTokenManagerInterface;
+use Tests\Base\HttpTestCase;
+use Zenstruck\Foundry\Persistence\Proxy;
+
+class ProcedureMapSettingApiTest extends HttpTestCase
+{
+    protected Procedure|Proxy|null $procedure;
+    protected ProcedureSettings|Proxy|null $procedureMapSetting;
+    protected User|Proxy|null $user;
+
+    public function testProcedureMapSetting(): void
+    {
+        $this->user = $this->getUserReference(LoadUserData::TEST_USER_2_PLANNER_ADMIN);
+        $this->procedure = ProcedureFactory::createOne();
+        $this->procedureMapSetting = ProcedureSettingsFactory::createOne();
+        $this->procedureMapSetting->setProcedure($this->procedure->_real());
+        $this->procedureMapSetting->_save();
+
+        $this->tokenManager = $this->getContainer()->get(JWTTokenManagerInterface::class);
+
+        $jwtToken = $this->initializeUser($this->user);
+        $headers = $this->getAdditionalHeaders($jwtToken, $this->procedure->_real());
+
```

Comment:
I would rather use `AbstractApiTest:sendRequest` in some ways, that handles all the work transparently and makes all the heavy lifting of User initialize and setting request headers

Reply:
Do you think it makes sense that HttpTestCase extends from JsonApiTest ? I think every request is usually checking for permissions if disable or enable.

Reply:
the JsonApiTest is very bound to api/2.0. I do not think that any HttpTestCase will always be api/2.0 as it is also meant to test simple Actions

Reply:
yes, actually, I think it would be very useful to test the whole page to check if things work instead of testing a resource type.